src/docs/product/issues/issue-details/index.mdx

```diff
@@ -29,6 +30,16 @@ There are two categories of issues: [error issues](/product/issues/issue-details
 Learn more about what's included an issue detail for each of these in [Error Issues](/product/issues/issue-details/error-issues/) and [Performance Issues](/product/issues/issue-details/performance-issues/).
 In addition, this page provides several other key pieces of information.
 
+## Event Dropdown
+
+By default, the issues page will display the "Recommended" event. To view other events, you can used the dropdown to skip to the latest or oldest event, or view the full list of events. If you wish to change the default event, you may do so in your [User Settings](https://sentry.io/settings/account/details/#defaultIssueEvent).
```

Comment:
```suggestion

```

Merged this into the next paragraph and moved the view/change stuff to after. I think this way, it's a bit shorter to read and flows better with all the recommended event info in the same place.